### PR TITLE
docs(README): GitHub Enterprise Server usage

### PR DESCRIPTION
**Pull request by leandredasilva**
The example provided to set an enterprise server URL doesn't work, 
It works with Octokit core defaults (tested) so I suggest this change in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,8 +235,6 @@ new OAuthApp({
 });
 ```
 
-Defaults to `@octokit/oauth-app`'s owne `Octokit` constructor which can be imported separately from `OAuthApp`. It's [`@octokit/core`](https://github.com/octokit/core.js) with the [`@octokit/auth-oauth-user`](https://github.com/octokit/auth-oauth-user.js/#readme) authentication strategy.
-
```
Comment (gr2m) [inline README.md:239]: this part is actually correct, can revert this change please?

**Review by gr2m — CHANGES_REQUESTED** [review]
Thanks a  lot for the pull request, that must have been a copy & paste fail. 

Can you please revert the 2nd change though?

**Review by gr2m — APPROVED** [review]
Thanks a lot!

File: README.md (hunk 1 above)
Comment (leandredasilva) [reply README.md]: Hello @gr2m, I have reverted the change, as requested.
